test/integ_test/evaluator/docker_eval/test_docker_evaluation.py

```diff
@@ -56,3 +59,17 @@ def test_evaluate_model(self, model_cls, model_config, metric, expected_res):
         for sub_type in metric.sub_types:
             joint_key = joint_metric_key(metric.name, sub_type.name)
             assert actual_res[joint_key].value >= expected_res
+
+    @patch("olive.systems.docker.docker_system.docker_utils.create_evaluate_command")
```

Comment:
I used the `patch` in integration test. Please let me know if you insist to keep no any patch in integration tests.

 https://stackoverflow.com/questions/52107522/is-it-in-considered-a-good-practice-to-mock-in-integration-test

Reply:
Yeah I don't quite like mock in integ test. This test is an unit test rather an integ test. What is the purpose of this test?

Reply:
whenever we use patch, it is something unit test.

Reply:
https://en.wikipedia.org/wiki/Mock_object

![image](https://github.com/microsoft/Olive/assets/817030/506cfd98-7176-4224-b4b1-2ee985d90c0c)

Reply:
Removed.